starter-code/lib/lodashexercise.js

```diff
@@ -0,0 +1,28 @@
+var opinions = ["This is the best job ever",
+  "Satisfied",
+  "At least I get paid",
+  "I'm looking for another job",
+  "I don't want to answer"
+];
+var sampleOpinions = "a";
+
+function functionName() {
+  sampleOpinions = _.sample(opinions);
+  return sampleOpinions;
+
+}
+
+var arrayName = [];
+
+function createArrayName() {
+  arrayName.push(_.times(10, functionName));
```

Comment:
La función `times` devuelve un array, por lo que estáis haciendo un push y poniendo un array sobre la primera posición de **arrayName**. No salen los nombres, por lo tanto.
Sabiendo que devuelve un array, sólo haría falta que lo igualáseis a otro array vacío:
`arrayName = _.times(10, functionName)`